app/controllers/openid_connect/token_controller.rb

```diff
@@ -1,5 +1,7 @@
 module OpenidConnect
   class TokenController < ApplicationController
+    prepend_before_action :skip_session_load
```

Comment:
Was this change intended to be here?

Reply:
Yes, but secretly. I accidentally left it out of #5170.

Reply:
Split into https://github.com/18F/identity-idp/pull/5177